11_developer_primer/3_developing_for_dynamo/13-dynamo-integration.md

```diff
@@ -388,19 +384,17 @@ Trace is a mechanism in dynamo core - it utilizes a static variable of callsites
 
 It also allows you to serialize arbitrary data into the .dyn file when writing zero touch dynamo nodes. This is not generally advisable as it means the potentially transferable zero touch code now has a dependency on dynamo core.
 
-Do not rely on the serialized format of the data in the .dyn file - instead use the \[Serializable] attribute and interface
```

Comment:
you should not remove this completely. It's still true as I commented above that the serialized data is not to be read directly. It should be accessed through the trace apis.